app/models/patient.rb

```diff
@@ -144,6 +144,19 @@ def still_urgent?
     false
   end
 
+  def contacted_since(datetime)
```

Comment:
You'll want to make this a class method instead of an instance method -- was just discussing this in #709 as a matter of fact! Also check out http://culttt.com/2015/06/10/understanding-class-methods-verses-instance-methods-in-ruby/ for a fuller explanation.